Makefile

```diff
@@ -196,17 +197,34 @@ build-image:
 run: build-image
 	@CID=$(shell docker run --net=host -d --name $(SERVER) $(IMAGE)) && echo $${CID}
 
+test-cli:
+	@for pkg in $(CLI_TEST_PACKAGES) ; do \
+		go test $$pkg ; \
+	done
+
 test-unit:
 	@for pkg in $(UNIT_TEST_PACKAGES) ; do \
 		go test $$pkg ; \
 	done
 
 test-integration:
+	@docker service rm amp-integration-test || true
+	@docker build -f Dockerfile.test -t appcelerator/amp-integration-test .
+	@docker service create --network amp-infra --name amp-integration-test --restart-condition none appcelerator/amp-integration-test
+	@n=""; \
+	while [[ $${n} == "" ]] ; do \
+		n=`docker ps -qf 'name=amp-integration'`; \
```

Comment:
@ndegory `docker service ps amp-integration-test` returns the taks id, and I need the container id to get the logs. Here is how it works currently:
 - starts the `amp-integration-test` service
 - get the container id
 - `docker logs -f $CONTAINER_ID` to get the output of the tests, which is cool because it's blocking and returns when the test ends.
 - get the exit code fo the container and exit with this code in the makefile to report the test exit status

Hopefuly, it's clearer with the explanation.